perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -2,12 +2,17 @@
 import uuid
 
 from autoware_auto_perception_msgs.msg import (
-    DetectedObject,
-    DetectedObjects,
-    ObjectClassification,
-    TrackedObject,
-    TrackedObjects,
+    ObjectClassification as AutowareAutoObjectClassification,
 )
+from autoware_auto_perception_msgs.msg import DetectedObject as AutowareAutoDetectedObject
+from autoware_auto_perception_msgs.msg import DetectedObjects as AutowareAutoDetectedObjects
+from autoware_auto_perception_msgs.msg import TrackedObject as AutowareAutoTrackedObject
+from autoware_auto_perception_msgs.msg import TrackedObjects as AutowareAutoTrackedObjects
+from autoware_perception_msgs.msg import DetectedObject as AutowareDetectedObject
+from autoware_perception_msgs.msg import DetectedObjects as AutowareDetectedObjects
+from autoware_perception_msgs.msg import ObjectClassification as AutowareObjectClassification
+from autoware_perception_msgs.msg import TrackedObject as AutowareTrackedObject
+from autoware_perception_msgs.msg import TrackedObjects as AutowareTrackedObjects
```

Comment:
According to updated `README`, users can install one of `autoware_auto_perception_msgs` or `autoware_perception_msgs`. Then will `ImportError` be caused if users do it?

Reply:
Sorry the readme was misleading. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/110/commits/436c6ae4b57f8d576b84b6bc09dde7da86301f87

(The basic idea is to install both libraries from build_depend.repos, but it either of the package already exists in the user's environment, the may remove either of them to build only the necessary one)